test/main.ts

```diff
@@ -733,3 +733,26 @@ test('parseJson option with promise.json() shortcut', async t => {
 
 	await server.close();
 });
+
+test('stringifyJson option with request.json()', async t => {
+	const server = await createHttpTestServer({bodyParser: false});
+
+	const json = {hello: 'world'};
+	const result = {data: json, extra: 'extraValue'};
+
+	server.post('/', async (request, response) => {
+		const body = await parseRawBody(request);
+		t.is(body, JSON.stringify(result));
+		response.end();
+	});
+
+	await ky.post(server.url, {
+		stringifyJson: data => JSON.stringify({
+			data,
+			extra: 'extraValue',
+		}),
```

Comment:
I think you misunderstood. `JSON.stringify({
			data,
			extra: 'extraValue',
		})` should be used both here and in 745.

Reply:
No, the `JSON.stringify` here is to add fields to any additional request data, not just the data in the example, so it's not appropriate to unify the two.